src/confluent_kafka/src/Admin.c

```diff
@@ -392,7 +392,8 @@ static PyObject *Admin_create_topics (Handle *self, PyObject *args,
         rd_kafka_NewTopic_t **c_objs;
         rd_kafka_queue_t *rkqu;
         CallState cs;
-
+        int default_partitions = 3;
+        int default_replications = 1;
```

Comment:
the default values don't come from the client, they are new values on the broker.